llama_toolchain/inference/api_instance.py

```diff
@@ -12,6 +12,10 @@ async def get_inference_api_instance(config: InferenceConfig):
         from .inference import InferenceImpl
 
         return InferenceImpl(config.impl_config)
+    elif config.impl_config.impl_type == ImplType.ollama.value:
+        from .inference import OllamaInference
```

Comment:
looks to be a typo. this should be `from .ollama`

Reply:
nice catch, updated the test to use this api.